### PR TITLE
S1 task3 add styling

### PR DESCRIPTION
**Pull request by cfbuckley14**
added styling to the application, slight update on database due to typo also:-)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 DS_Store
```
Comment (maxwell-01) [inline .gitignore]: you can delete this

File: index.php [hunk 2]
```diff
@@ -11,15 +11,23 @@
...
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300&display=swap" rel="stylesheet">
+</head>
 
-    <div>
-        <h1>Marvel Cinematic Universe Characters</h1>
+<body>
+    <div class="heading">
```
Comment (maxwell-01) [inline index.php]: remove the div and class and put into <header> tags

File: index.php [hunk 3]
```diff
@@ -11,15 +11,23 @@
...
+        <h1>MARVEL CHARACTERS</h1>
     </div>
-
-    <div>
+    <div class="parentContainer">
```
Comment (maxwell-01) [inline index.php]: once the above is in the header you don't need extra divs here as they don't do anything extra

Comment (maxwell-01) [reply index.php]: you can use <main> tags

File: style.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    background-color: #b13536;
+    height: 300px;
+    width: 800px;
+    font-size: 15rem;
+    text-align: center;
+    padding: 20px;
+    margin: 0;
+}
+
+.heading {
```
Comment (maxwell-01) [inline style.css]: this should be `header`

**Review by maxwell-01 — APPROVED** [review]
couple of comments but looks really good!

**Review by RichVelo — APPROVED** [review]
the changes look to have been made - nice work

**Pull request merged**